packages/app-runtime/test/modules/DatawalletSynchronizedModule.test.ts

```diff
@@ -0,0 +1,158 @@
+            return;
+        }
+
+        let abortResult;
+        if (activeIdentityDeletionProcess.value.status === IdentityDeletionProcessStatus.Approved) {
+            abortResult = await sessionDevice1.transportServices.identityDeletionProcesses.cancelIdentityDeletionProcess();
+            await sessionDevice2.transportServices.account.syncDatawallet();
+            // await TestUtil.awaitEvent(runtimeDevice2, LocalAccountDeletionDateChangedEvent);
+        }
+        if (abortResult?.isError) throw abortResult.error;
+    });
+
+    afterAll(async function () {
+        await runtimeDevice1.stop();
+        await runtimeDevice2.stop();
+    });
+
+    test("should set the deletionDate of the LocalAccount initiate an IdentityDeletionProcess on a second device that is online", async function () {
+        const initiateResult = await sessionDevice1.transportServices.identityDeletionProcesses.initiateIdentityDeletionProcess();
+        expect(sessionDevice2.account.deletionDate).toBeUndefined();
+
+        await sessionDevice2.transportServices.account.syncDatawallet();
+        await TestUtil.awaitEvent(runtimeDevice2, LocalAccountDeletionDateChangedEvent);
+
+        expect(sessionDevice2.account.deletionDate).toBeDefined();
+        expect(sessionDevice2.account.deletionDate).toBe(initiateResult.value.gracePeriodEndsAt);
+    });
+
+    test("should unset the deletionDate of the LocalAccount cancelling an IdentityDeletionProcess on a second device that is online", async function () {
+        await sessionDevice1.transportServices.identityDeletionProcesses.initiateIdentityDeletionProcess();
+        await sessionDevice2.transportServices.account.syncDatawallet();
+        await TestUtil.awaitEvent(runtimeDevice2, LocalAccountDeletionDateChangedEvent);
+
+        await sessionDevice1.transportServices.identityDeletionProcesses.cancelIdentityDeletionProcess();
+        expect(sessionDevice2.account.deletionDate).toBeDefined();
+
+        await sessionDevice2.transportServices.account.syncDatawallet();
+        await TestUtil.awaitEvent(runtimeDevice2, LocalAccountDeletionDateChangedEvent);
+
+        expect(sessionDevice2.account.deletionDate).toBeUndefined();
+    });
+
+    test("should set the deletionDate of the LocalAccount initiating an IdentityDeletionProcess on a second device that was offline", async function () {
+        await runtimeDevice2.stop();
+
+        await sessionDevice1.transportServices.identityDeletionProcesses.initiateIdentityDeletionProcess();
+        expect(sessionDevice2.account.deletionDate).toBeUndefined();
+
+        await runtimeDevice2.init(); // TODO: do the modules need to be turned on?
+        await runtimeDevice2.start();
+
+        await sessionDevice2.transportServices.account.syncDatawallet();
+        // await TestUtil.awaitEvent(runtimeDevice2, LocalAccountDeletionDateChangedEvent);
+
+        expect(sessionDevice2.account.deletionDate).toBeDefined();
```

Comment:
@jkoenig134 Here we don't seem to go into the DatawalletSynchronized module. Do you have any idea why?